src/parser.ts

```diff
@@ -344,6 +371,14 @@ export class Parser {
      * Get the current elements as represented by the builder
      */
     public result(): Elements {
+        if (this.byteParser.current instanceof InDeflated) {
+            const inDeflatedStep = (this.byteParser.current as InDeflated)
+            this.byteParser.startWith(new InAttribute(inDeflatedStep.state, inDeflatedStep.stop));
+            const buff = inDeflatedStep.inflate();
+            this.byteParser.parse(buff);
+            this.byteParser.isCompleted = true;
+        }
+        const elements = this.builder.build();
```

Comment:
This line should not be necessary? We're returning `this.builder.build()` just underneath.